main.php

```diff
@@ -117,6 +117,8 @@
 require_once('core/startup/ErrorControlChain.php');
 require_once('core/startup/ParameterConfirmationToken.php');
 
+ini_set('display_errors', 0);
```

Comment:
For instance, a YAML error might make the logging configuration broken, in which case I'd expect an error to be displayed.